packages/v2/gui/src/sak/behandling-velger/components/BehandlingPicker.tsx

```diff
@@ -53,8 +52,12 @@ const getSøknadsperioderForValgtBehandling = (
   valgtBehandling?: Behandling,
 ) => {
   const dataForValgtBehandling = søknadsperioder.find(periode => periode.data?.id === valgtBehandling?.id)?.data;
-
-  if (valgtBehandling?.visningsnavn === KONTROLL_AV_INNTEKT_VISNINGSNAVN) {
+  const erKontrollAvInntekt = valgtBehandling?.behandlingÅrsaker?.every(
```

Comment:
Dette blir nesten riktig.
Bortsett fra at hvis `behandlingÅrsaker` også inneholder `UTTALELSE_FRA_BRUKER` eller `RE_REGISTEROPPLYSNING` så vil ikke denne filtreringen fungere.

Da er det bedre å filtrere ut disse to typene først før du utleder kontroll av inntekt.

Men da er jeg redd vi introduserer backend logikk i frontend. 
Som det var før er det nok bedre fordi backend gjør denne filtreringen, men kanskje gå bort fra string som visningsnavn og heller bruke enum. Hva tenker du?